File: src/main/java/com/manifestors/shinrai/client/module/modules/visuals/NoFOV.java
package com.manifestors.shinrai.client.module.modules.visuals;

import com.manifestors.shinrai.client.module.Module;
import com.manifestors.shinrai.client.module.ModuleCategory;
import com.manifestors.shinrai.client.module.annotations.ModuleData;
import org.lwjgl.glfw.GLFW;

@ModuleData(
        name = "NoFOV",
        description = "Disables FOV camera effect.",
        category = ModuleCategory.VISUALS,
        keyCode = GLFW.GLFW_KEY_V
)
public class NoFOV extends Module {
}

File: src/main/kotlin/com/manifestors/shinrai/client/module/modules/visuals/NoFOV.kt
package com.manifestors.shinrai.client.module.modules.visuals

import com.manifestors.shinrai.client.module.Category
import com.manifestors.shinrai.client.module.Module
import org.lwjgl.glfw.GLFW

class NoFOV : Module(
    name = "NoFOV",
    description = "Disables dynamic FOV effect.",
    category = Category.VISUALS,
    key = GLFW.GLFW_KEY_V
)